lib/KToolbar.vue

```diff
@@ -1,328 +0,0 @@
-    name: 'KToolbar',
-    components: {
-      UiIconButton,
-      UiProgressLinear,
-    },
-
-    props: {
-      /**
-       * Controls the toolbar appearance and styling. Options are:
-       * - 'default': Standard white background with shadow
-       * - 'colored': Uses theme primary color as background
-       * - 'clear': Transparent background with no shadow or border
-       */
-      type: {
-        type: String,
-        default: 'default',
-        validator(value) {
-          return ['default', 'colored', 'clear'].includes(value);
-        },
-      },
-      /**
-       * Controls the text color throughout the toolbar. Use 'white' for dark backgrounds
-       * and 'black' for light backgrounds to ensure proper contrast.
-       */
-      textColor: {
-        type: String,
-        default: 'black',
-        validator(value) {
-          return ['black', 'white'].includes(value);
-        },
-      },
-      /**
-       * The main title text displayed in the toolbar
-       */
-      title: {
-        type: String,
-        default: '',
-      },
-      /**
-       * Brand text displayed alongside the title in the left area
-       */
-      brand: {
-        type: String,
-        default: '',
-      },
-      /**
-       * Icon name displayed in the navigation button when using the default icon slot
-       */
-      navIcon: {
-        type: String,
-        default: 'menu',
-      },
-      /**
-       * Whether to hide the navigation icon button completely
-       */
-      removeNavIcon: {
-        type: Boolean,
-        default: false,
-      },
-      /**
-       * Whether the toolbar has elevated shadow styling for visual hierarchy
-       */
-      raised: {
-        type: Boolean,
-        default: true,
-      },
-      /**
-       * Position of the loading progress bar indicator. Options are 'top' or 'bottom'.
-       */
-      progressPosition: {
-        type: String,
-        default: 'bottom',
-        validator(value) {
-          return ['top', 'bottom'].includes(value);
-        },
-      },
-      /**
-       * Whether to show the loading progress indicator
-       */
-      loading: {
-        type: Boolean,
-        default: false,
-      },
-    },
-
-    computed: {
-      classes() {
-        return [
-          `k-toolbar--type-${this.type}`,
-          `k-toolbar--text-color-${this.textColor}`,
-          `k-toolbar--progress-position-${this.progressPosition}`,
-          { 'is-raised': this.raised },
-        ];
-      },
-      toolbarStyles() {
-        if (this.type === 'colored') {
-          return { backgroundColor: this.$themeBrand.primary.v_600 };
-        }
-        return {};
-      },
-      progressColor() {
-        return this.textColor === 'black' ? 'primary' : 'white';
-      },
-    },
-
-    methods: {
-      navIconClick() {
-        /**
-         * Emitted when the navigation icon is clicked
-         * @event nav-icon-click
-         */
-        this.$emit('nav-icon-click');
```

Comment:
Removed. Since `navIcon` and the default `UiIconButton` was removed, this will never be emitted. 

**Audit**

- Used in Kolibri from [ImmersiveToolbar](https://github.com/learningequality/kolibri/blob/ad8ebbdc43144f36d3d57d66d0e88f8c094d6853/packages/kolibri/components/pages/ImmersivePage/internal/ImmersiveToolbar.vue#L13) but doesn't take effect (since we don't use `navIcon`, but instead use the `#icon` slot and [emit `nav-icon-click` from `KIconButton`](https://github.com/learningequality/kolibri/blob/ad8ebbdc43144f36d3d57d66d0e88f8c094d6853/packages/kolibri/components/pages/ImmersivePage/internal/ImmersiveToolbar.vue#L38-L54))